homeassistant/components/sharkiq/config_flow.py

```diff
@@ -11,14 +11,20 @@
 import voluptuous as vol
 
 from homeassistant import config_entries, core, exceptions
-from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.const import CONF_PASSWORD, CONF_REGION, CONF_USERNAME
 from homeassistant.data_entry_flow import FlowResult
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 
 from .const import DOMAIN, LOGGER
 
 SHARKIQ_SCHEMA = vol.Schema(
-    {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_REGION, default="Everywhere Else"): vol.In(
+            ["Europe", "Everywhere Else"]
+        ),
```

Comment:
I think there are ways to use translations in config-flow, but I'm not quite sure.
Regardless I'd probably use a code/description structure rather than just plain text.

Reply:
This hasn't really been adressed. I think you should leave it open.
Maybe another reviewer has some advice.

Reply:
Added translation support.  Is that along the lines of what you were thinking?